### DISCUSSION TITLE
Regex

### DISCUSSION THREAD

**Discussion by hoangtraann** [community/community#65156] [Programming Help]
### Body

help me write regex that have at least 1 uppercase word, 1 lowercase word and 1 number (accept symbol)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by nhh0718** [comment]
Here you are: 
^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).+$

**Comment by queenofcorgis** [comment]
@hoangtraann @nhh0718  GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
